sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyAlertConfigurationLiveTests.cs

```diff
@@ -19,9 +19,11 @@ public AnomalyAlertConfigurationLiveTests(bool isAsync) : base(isAsync)
         }
 
         [RecordedTest]
-        public async Task CreateAndGetAlertConfigurationWithWholeSeriesScope()
+        [TestCase(true)]
+        [TestCase(false)]
```

Comment:
If this variation will be needed for every test, consider using the `ClientTestFixture` attribute with `additionalParameters` configured. 

[Here is an example](https://github.com/Azure/azure-sdk-for-net/blob/0f8dada4e59733da7b8fa15b530d68edf4ccfaa7/sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs#L14) of using it with Tables to create an additional permutation of each test to target both Cosmos and Storage endpoints.

Reply:
We're only adding the parameters in one test for each endpoint. We just need to make sure that AAD works, so we're not that concerned about validating every single possibility. We do the complete validation with the key credential only.